src/features/rewards/walletWrapper/index.tsx

```diff
@@ -113,6 +163,62 @@ export default class WalletWrapper extends React.PureComponent<Props, State> {
     return grants && grants.length > 0
   }
 
+  hasNotifications = (notifications?: Notification[]) => {
+    return notifications && notifications.length > 0
+  }
+
+  getNotificationIcon = (type: NotificationType) => {
+    let icon
+
+    switch (type) {
+      case 'ads':
+        icon = <MegaphoneIcon />
+        break
+      case 'contribute':
+        icon = <LoveIcon />
+        break
+      case 'token':
+        icon = <GiftIcon />
+        break
+      default:
+        icon = ''
```

Comment:
in this case I think case we should return null. This way we don't have 58x58 big empty block